### PR TITLE
Adds new integration [domalab/ha-unraid]

### PR DESCRIPTION
**Pull request by domalab**
## Checklist

- [x] I've read the [publishing documentation](https://hacs.xyz/docs/publish/start).
- [x] I've added the [HACS action](https://hacs.xyz/docs/publish/action) to my repository.
- [x] (For integrations only) I've added the [hassfest action](https://developers.home-assistant.io/blog/2020/04/16/hassfest/) to my repository.
- [x] The actions are passing without any disabled checks in my repository.
- [x] I've added a link to the action run on my repository below in the links section.
- [x] I've created a new release of the repository after the validation actions were run successfully.

## Links

Link to current release: [https://github.com/domalab/ha-unraid/releases/tag/v0.1.1](url)
Link to successful HACS action (without the `ignore` key): [https://github.com/domalab/ha-unraid/actions/runs/11358689322](url)
Link to successful hassfest action (if integration): [https://github.com/domalab/ha-unraid/actions/runs/11358689333](url)

### CODE REVIEW

File: integration [hunk 1]
```diff
@@ -280,6 +280,8 @@
   "dmamontov/hass-seafile",
   "dmoranf/home-assistant-wattio",
   "dolezsa/thermal_comfort",
+  "domalab/ha-actronair-neo",
```
Comment (ludeeus) [inline integration:283]: Limit each PR to a single entry.
Your supplied links are not valid.

**Review by ludeeus — CHANGES_REQUESTED** [review]

File: integration (hunk 1 above)
Comment (domalab) [reply integration]: Ok. I removed the domalab/ha-actronair-neo integration until my UNRAID one gets approved then I'll do the other one.

**Review by domalab — COMMENTED** [review]
Removed ActronAir Neo entry until UNRAID integration gets approved

File: integration (hunk 1 above)
Comment (ludeeus) [reply integration]: If you look at the size of the queue, you might want to reconsider that and do both as separate now ;)
Hint: do not use the "master" branch of your fork..
https://hacs.xyz/docs/publish/include/#before-submitting

**Pull request closed without merging**